jekyll-export-cli.php

```diff
@@ -16,12 +16,54 @@
  *
  * Must be run in the wordpress-to-jekyll-exporter/ directory.
  */
-require '../../../wp-load.php';
+
+// Uncomment for extra replace options
+
+$wordpress_path = "../../../";
+//$wordpress_path = "/home/wordpress/";
+
+require $wordpress_path . 'wp-load.php';
 require_once 'jekyll-exporter.php'; // Ensure plugin is "activated".
 
+$config["site_url"] = "https://www.mywebsite.fr/";
+
+$config["extra_remove_site_url"] = $config["site_url"];  // Will remove site url to get relative URLs
+$config['extra_code'] = true; // Will retrieve lang and protect pre/code
+$config['extra_img_style'] = true; // Will retrieve style from image
+$config['extra_tag_spoiler'] = true; // Will convert spoiler tags
+
+//$config['force_dest_dir'] = "/dest-path/"; // "_export/"; // Force destination directory to relative one (without temp folder, will deactivate zip)
+
+
 if ( php_sapi_name() !== 'cli' ) {
 	wp_die( 'Jekyll export must be run via the command line or administrative dashboard.' );
 }
 
+add_filter( 'the_content', function($contents) {
```

Comment:
Is there any advantage to making this function a named callback? Also, would this behavior then only work for the CLI implementation, and not the web interface? Could we move it into shared logic?

Reply:
True, moved in the shared file and named function.